### ISSUE TITLE
SEO Fill: 15 tool(s) need metadata

### ISSUE THREAD

**Issue by linisme** [nologin-tools/nologin.tools#249]
@claude Please generate SEO metadata for the tools listed below.

# SEO Fill: 15 tool(s) need SEO metadata

Create one JSON file per tool at `.github/auto-seo/{slug}.json`.

## SEO Writing Standards

For each tool, generate a JSON file `.github/auto-seo/{slug}.json` with these fields:

| Field | Constraint | Guidance |
|-------|-----------|----------|
| `seoTitle` | **≤ 44 chars** | The site appends " \| nologin.tools" (16 chars), so Google's ~60-char display = 44 for your title. Format: "{Name} — {core value}" or "{Name}: Free Online {type}". Do NOT append "No Login Required". |
| `seoDescription` | **120-155 chars** | Unique, compelling copy. Must naturally mention no-login/no-signup. Do NOT start with "{Name} helps you...". Include a clear benefit or CTA. |
| `seoFocusKeyword` | **2-5 word phrase** | A realistic search query users would type, e.g. "free online whiteboard no signup". |
| `seoIntent` | **task / alternative / review** | `task` = user wants to do something; `alternative` = user is looking for a free/no-login alternative to a paid/login tool (e.g. Photopea as Photoshop alternative); `review` = user is evaluating the tool. |
| `seoTaskPhrase` | **≤ 60 chars, must contain no-login reference** | A concise action phrase like "draw diagrams online without signup". Must include one of: "without login", "without signup", "no login", "no signup", "no account", "without registration", "no registration". |

### JSON format per tool

```json
{
  "seoTitle": "Excalidraw — Free Online Whiteboard",
  "seoDescription": "Draw hand-sketched diagrams and wireframes right in your browser. Real-time collaboration, PNG/SVG export — no signup needed.",
  "seoFocusKeyword": "excalidraw online whiteboard free",
  "seoIntent": "task",
  "seoTaskPhrase": "draw diagrams collaboratively without signup"
}
```

### Quality rules
- Every `seoTitle` must be **unique** across all tools
- Every `seoDescription` must be **unique** — no two tools share the same description
- Do NOT use truncated text ending with "..."
- Do NOT repeat "without login" twice in any field
- `seoDescription` must NOT start with "{ToolName} helps you" or "{ToolName} is a"
- Write in English

---

### DeepL Translator (`deepl-com-translator`)
- **URL**: https://deepl.com/translator
- **Description**: AI-powered translation engine with superior accuracy across 30+ languages. Free web version translates up to 1,500 characters per request.
- **Core Task**: Translate text between languages with AI accuracy
- **Tags**: category:AI, pricing:Freemium, hosting:Cloud Only, offline:Online Only, data:Server-Side, type:Web App, source:Closed Source

### QuillBot (`quillbot-com`)
- **URL**: https://quillbot.com
- **Description**: AI paraphrasing tool that rewrites text for clarity and style. Free version offers Standard and Fluency modes with unlimited paraphrases, no signup required.
- **Core Task**: Paraphrase and rewrite text with AI
- **Tags**: category:AI, pricing:Freemium, hosting:Cloud Only, offline:Online Only, data:Server-Side, type:Web App, source:Closed Source

### Craiyon (`craiyon-com`)
- **URL**: https://craiyon.com
- **Description**: Free AI image generator that creates 9 images from text prompts. No login required, unlimited generations, commercial use allowed with credit.
- **Core Task**: Generate images from text descriptions with AI
- **Tags**: category:AI, pricing:Ad-Supported, hosting:Cloud Only, offline:Online Only, data:Server-Side, type:Web App, source:Closed Source

### Teachable Machine (`teachablemachine-withgoogle-com`)
- **URL**: https://teachablemachine.withgoogle.com
- **Description**: Google's no-code tool for training machine learning models in the browser. Train image, sound, or pose recognition models with your own data, all processed client-side.
- **Core Task**: Train machine learning models without coding
- **Tags**: category:AI, pricing:Free, hosting:Cloud Only, offline:Online Only, data:Client-Side Only, type:Web App, source:Open Source, privacy:Privacy Focused

### TensorFlow Playground (`playground-tensorflow-org`)
- **URL**: https://playground.tensorflow.org
- **Description**: Interactive neural network visualization in the browser. Experiment with layers, neurons, activation functions, and learning rates to understand how neural networks learn.
- **Core Task**: Visualize and experiment with neural networks interactively
- **Tags**: category:AI, pricing:Free, hosting:Self-Hostable, offline:Works Offline, data:Client-Side Only, type:Web App, source:Open Source

### Goblin.tools (`goblin-tools`)
- **URL**: https://goblin.tools
- **Description**: AI-powered task management collection designed for neurodivergent users. Magic ToDo breaks overwhelming tasks into manageable steps with adjustable complexity. Free forever, no ads.
- **Core Task**: Break down complex tasks into manageable steps with AI
- **Tags**: category:AI, pricing:Free, hosting:Cloud Only, offline:Online Only, data:Server-Side, type:Web App, source:Closed Source

### AskMeAnything (`askmeanythi-ng`)
- **URL**: https://askmeanythi.ng
- **Description**: The simplest way to host an AMA session. Create a space, share the link, and start answering. No registration required.
- **Core Task**: Create and host AMA sessions, share links, ask and answer questions
- **Tags**: category:Communication, source:Open Source, data:Server-Side, type:Web App, hosting:Self-Hostable, offline:Online Only, pricing:Free

### Cookies Extractor (`cookies-extractor-chekun-me`)
- **URL**: https://cookies-extractor.chekun.me/
- **Description**: A simple yet powerful Chrome extension that helps you extract and export cookies from any website with just one click.
- **Core Task**: Extract Cookies
- **Tags**: category:Development, source:Open Source, data:Client-Side Only, privacy:No Trackers, privacy:Privacy Focused, type:Browser Extension, offline:Works Offline, pricing:Free

### SiteAge (`siteage-org`)
- **URL**: https://siteage.org
- **Description**: Trace the origins of any website through the Internet Archive. Look up any domain to receive a certified age badge with birth date, longevity ranking, historical timeline, and embeddable SVG badges for your website or README.
- **Core Task**: Look up any domain's age, view historical timeline, and get embeddable SVG age badges
- **Tags**: category:Data, source:Open Source, data:Server-Side, privacy:No Trackers, privacy:Privacy Focused, type:Web App, hosting:Cloud Only, offline:Online Only, pricing:Free

### next-tools (`next-tools-dev`)
- **URL**: https://next-tools.dev
- **Description**: This is a tool website forked from it-tools, containing 100+ tools.
- **Core Task**: all the things
- **Tags**: category:Development, data:Client-Side Only, privacy:Privacy Focused, type:PWA, hosting:Self-Hostable, offline:Works Offline, pricing:Free, source:Open Source

### All-in-one tool for image cropping, collages, photo walls, and poster creation. (`ps-keepmot-com`)
- **URL**: https://ps.keepmot.com/
- **Description**: A free online collage and image editing tool. No login required, completely free to use. Built entirely on the front end, ensuring absolute privacy and security for your images. Download high-resolution results with no watermarks. A clean and professional tool for creating beautiful visual works.
- **Core Task**: 1. Grid Collage 2. Photo Wall Collage 3. Long Image Stitching 4. Image Cropping 5. Image Effects
- **Tags**: category:Design, data:Client-Side Only, privacy:No Trackers, type:Browser Extension, hosting:Cloud Only, offline:Online Only, pricing:Free

### Kuakua - Psychology & AI Tools for Well-Being (`kuakua-app`)
- **URL**: https://kuakua.app/
- **Description**: Kuakua.app is your AI-powered hub for mental health resources, mindfulness meditation, and positive psychology games. Take science-backed quizzes, play brain-boosting tasks, and unlock daily affirmations to feel happier, calmer, and more resilient—anytime, on any device.
- **Core Task**: - **Comprehensive Global Collection of Psychology Resources:**You can find all the things you want in psychology here. - **Experiential Psychology Content:** Aims to provide various experiential psych
- **Tags**: category:Education, data:Client-Side Only, type:Web App, offline:Online Only, pricing:Free

### PhET Interactive Simulations (`phet-colorado-edu`)
- **URL**: https://phet.colorado.edu
- **Description**: Free browser-based science and math simulations from the University of Colorado Boulder. Over 150 interactive HTML5 simulations covering physics, chemistry, biology, earth science, and math — all usable directly in the browser with no account required.
- **Core Task**: Run interactive STEM simulations in the browser to explore science and math concepts visually
- **Tags**: category:Education, pricing:Free, type:Web App, data:Client-Side Only, hosting:Cloud Only, source:Open Source

### Phind (`phind-com`)
- **URL**: https://phind.com
- **Description**: An AI-powered search engine built for developers. Phind searches the web in real time and synthesizes technical answers with code examples, documentation references, and relevant sources — no account required for basic usage.
- **Core Task**: Get AI-generated answers to programming and technical questions with real-time web search and working code examples
- **Tags**: category:AI, pricing:Freemium, type:Web App, data:Server-Side

### Convertio.com (`convertio-com`)
- **URL**: https://convertio.com
- **Description**: Convert between 200+ formats with precision. Audio, video, images, documents — powered by professional-grade tools.
- **Core Task**: Drag file and convert it
- **Tags**: category:Data, data:Server-Side, hosting:Self-Hostable, offline:Online Only, pricing:Free

**Comment by linisme** [comment]
**Claude finished @linisme's task in 6m 11s** —— [View job](https://github.com/nologin-tools/nologin.tools/actions/runs/24859137253) • [`claude/issue-249-20260423-2113`](https://github.com/nologin-tools/nologin.tools/tree/claude/issue-249-20260423-2113) • [Create PR ➔](https://github.com/nologin-tools/nologin.tools/compare/main...claude/issue-249-20260423-2113?quick_pull=1&title=seo%3A%20add%20metadata%20for%2015%20tools%20(batch%20%23249)&body=Generates%20SEO%20metadata%20for%2015%20tools%3A%20DeepL%2C%20QuillBot%2C%20Craiyon%2C%20Teachable%20Machine%2C%20TensorFlow%20Playground%2C%20Goblin.tools%2C%20AskMeAnything%2C%20Cookies%20Extractor%2C%20SiteAge%2C%20next-tools%2C%20Keepmot%2C%20Kuakua%2C%20PhET%2C%20Phind%2C%20Convertio.%0A%0ACloses%20%23249%0A%0AGenerated%20with%20%5BClaude%20Code%5D(https%3A%2F%2Fclaude.ai%2Fcode))

---
### SEO Fill: 15 tools metadata generated

- [x] Analyze request and plan SEO metadata
- [x] Create `.github/auto-seo/` directory and generate all 15 JSON files
- [x] Commit and push changes
- [x] Create PR

All 15 JSON files created with unique `seoTitle`, `seoDescription`, `seoFocusKeyword`, `seoIntent`, and `seoTaskPhrase` fields — all within the required character limits and including no-login references.

| Slug | Title | Desc len |
|------|-------|----------|
| `deepl-com-translator` | DeepL Translator — AI Translation Online | 125 |
| `quillbot-com` | QuillBot — AI Paraphrasing Tool Free | 139 |
| `craiyon-com` | Craiyon — Free AI Image Generator | 135 |
| `teachablemachine-withgoogle-com` | Teachable Machine — Train ML Models Free | 151 |
| `playground-tensorflow-org` | TensorFlow Playground — Neural Net Viz | 154 |
| `goblin-tools` | Goblin.tools — AI Task Breakdown Free | 147 |
| `askmeanythi-ng` | AskMeAnything — Host AMA Sessions Free | 152 |
| `cookies-extractor-chekun-me` | Cookies Extractor — One-Click Cookie Export | 141 |
| `siteage-org` | SiteAge — Find Any Domain's Age Free | 149 |
| `next-tools-dev` | next-tools — 100+ Developer Utilities | 150 |
| `ps-keepmot-com` | Keepmot — Free Online Collage Maker | 151 |
| `kuakua-app` | Kuakua — AI Mental Wellness Platform | 155 |
| `phet-colorado-edu` | PhET Simulations — Free STEM Learning | 151 |
| `phind-com` | Phind — AI Developer Search Engine | 152 |
| `convertio-com` | Convertio — Convert 200+ File Formats | 138 |

**Comment by linisme** [comment]
✅ SEO metadata updated for 15 tool(s).
